requirements/env_omnisci.yml

```
@@ -13,7 +13,7 @@ dependencies:
   - pytest-xdist>=2.1.0
   - coverage<5.0
   - pygithub==1.53
-  - pyomniscidbe<=5.8
+  - pyomniscidbe==5.10.0
```

```suggestion
  - pyomniscidbe=5.10
```
this would allow 5.10.0, 5.10.1, etc - I guess this should suit us well

---

Not sure, @Garra1980, @ienkovich, thoughts?

---

I think yes, =5.10 is ok here

---

fixed